addons/bus/static/src/workers/websocket_worker.js

```diff
@@ -392,9 +397,12 @@ export class WebsocketWorker {
      * If the websocket is not open, enqueue the message and send it
      * upon the next reconnection.
      *
-     * @param {any} message Message to send to the server.
+     * @param {{event_name: string, data: any }} message Message to send to the server.
      */
     _sendToServer(message) {
+        if (Object.keys(this._context).length > 0) {
+            message.context = this._context;
+        }
```

Comment:
why not always set the context?

Reply:
Why should we add overhead to every request if there is nothing to send?